iot_api/user_api/events/alert_events.py

```diff
@@ -114,7 +114,21 @@ def handle_alert_events(ch, method, properties, body):
                 LOG.error(f"Error {e} on NotificationAssetImportance search for user {user.id}. Ignoring asset_importance preference")
                 is_important_for_user = True
 
-            if alert_settings and getattr(alert_settings, alert_type.risk.lower()) and is_important_for_user and dc_settings and dc_settings.enabled:
+            # Check whether the alert assets contain all the tags in user notification preferences or not
+            try:
+                asset_tags = NotificationAssetTag.find_all_by_user_id(user.id)
+                tag_id_list = []
+                for asset_tag in asset_tags:
+                    tag_id_list.append(asset_tag.tag_id)
```

Comment:
120-122 can be done with list comprehension:

tag_id_list = [asset_tag .tag_id for asset_tag in asset_tags]

It is faster and easier to read.

Reply:
Done